feedback/index.html

```diff
@@ -0,0 +1,352 @@
+
+    <img alt="Image of hands passing plywood through a bandsaw in SNTL Workspace" class="heroimage" src="images/scroll+saw.jpg">
+
+    <h2 class="aboutus">About Us
+
+    </h2>
+
+    <!-- Is there a better/ proper way to scale a decorative line? I used svg first and it was even less responsive. I used aria hidden to hise from screen reading but is there a backup if this isnt supported by browser?  -->
+
+    <div class="aboutus_stroke" aria-hidden="true"></div>
+
+    <section class="aboutus_copy">
+        <p class="aboutus_copy"> SNTL is a nonprofit tool-lending library and community hub in the heart of Baltimore
+            City. We offer
+            over
+            3,000
+            tools, 30 classes, a public woodworking shop, a dedicated DIY workspace, and a home repair classroom.</p>
+        <p>Our space is a welcoming environment where people can be creative, foster a do-it-yourself attitude, and
+            learn
+            to work with their hands. We share what we know and learn from each other!</p>
+    </section>
+
+    <section class="cta">
+        <a href="#" class="cta_link1">Learn</a>
+
+        <a href="#" class="cta_link2">Join</a>
+
+        <a href="#" class="cta_link3">Renew</a>
+    </section>
+
+    <h2 class="upcomingevents">Upcoming Events</h2>
+
+    <div class="upcomingevents_stroke" aria-hidden="true"></div>
+
+    <section class="upcomingevents_contents">
+        <ol>
+            <li>
+                <h4>Fall Fix-It Fair</h4>
+                <time>Oct 20, 2018</time>
+            </li>
+            <li>
+                <h4>October Open House</h4>
+                <time>Oct 26, 2018</time>
+            </li>
+            <li>
+                <h4>Tax Credits & Resources for Home Improvements</h4>
+                <time>November 8, 2018</time>
+            </li>
+        </ol>
+    </section>
+
+
+    <footer>
+
+        <h2 class="footer_sec1">Library Hours</h2>
+
+        <section class="footer_sec2">
+            <time>Monday - Thursday; 5pm-8pm</time><br>
+            <time>Saturday - Sunday; 11am - 5pm</time>
+        </section>
+
+        <h2 class="footer_sec3">Open Shop Hours</h2>
+        <section class="footer_sec4">
+            <time>Tuesdays; 2pm-8pm</time><br>
+            <time>Sundays; 11am - 5pm</time>
+        </section>
+
+
+        <h2 class="footer_sec5">Our Location</h2>
+        <a class="footer_sec6" href="https://www.google.com/maps/place/Station+North+Tool+Library/@39.3066755,-76.6122498,17z/data=!3m1!4b1!4m5!3m4!1s0x89c80493276aaa65:0x3b0cbffb9c347e16!8m2!3d39.3066755!4d-76.6100611?hl=en">417
+            E Oliver St. Baltimore 21202</a>
+
+
+        <h2 class="footer_sec7">Contact Us </h2>
+        <section class="footer_sec8">
+            <address>
+                410-347-0850
+            </address>
+            <address>
+                info@stationnorthtoollibrary.org
```

Comment:
Maybe wrap this in a [`mailto`](https://css-tricks.com/snippets/html/mailto-links/) link?